mteb/tasks/Retrieval/GermanDPRRetrieval.py

```diff
@@ -0,0 +1,56 @@
+from ...abstasks.AbsTaskRetrieval import AbsTaskRetrieval
+
+import datasets
+
+
+class GermanDPR(AbsTaskRetrieval):
+    _EVAL_SPLIT = "test"
+    _LANGUAGE = "de"
+
+    @property
+    def description(self):
+        return {
+            "name": "GermanDPR",
+            "hf_hub_name": "deepset/germandpr",
+            "description": "GermanDPR is a German Question Answering dataset for open-domain QA. It associates "
+            "questions with a textual context containing the answer",
+            "reference": "https://www.deepset.ai/germanquad",
+            "type": "Retrieval",
+            "category": "s2p",
+            "eval_splits": [self._EVAL_SPLIT],
+            "eval_langs": [self._LANGUAGE],
+            "main_score": "ndcg_at_10",
+            "revision": "5129d02422a66be600ac89cd3e8531b4f97d347d",
+        }
+
+    def _format_documents(self, docs, id_prefix=""):
+        result = {}
+        for i, (title, content) in enumerate(zip(docs["title"], docs["text"])):
+            id_value = f"{id_prefix}{i}"
+            formatted_content = content.split("===")[-1].replace("\n", " ").lstrip()
```

Comment:
Should the splitting text be changed to "==\n" or just left like this?

Reply:
Good point, this seems more robust, I will change it